core/lib/multistore.ts

```diff
@@ -217,3 +217,11 @@ export function localizedRoutePath (path: string, storeCode: string): string {
 
   return `/${storeCode}/${_path}`
 }
+
+export function localizedRouteOnlyForDefault (path: string) {
```

Comment:
I think this is a workaround only for this footr thing. I'd suggest solving this on theme level - this is misleading helper. It should be defined only in the `Category.vue`